enterprise/backend/src/metabase_enterprise/metabot_v3/envelope.clj

```diff
@@ -97,10 +97,14 @@
 
 (defn- message->reaction
   [msg]
-  {:type :metabot.reaction/message
-   :repl/message-color :green
-   :repl/message-emoji "🤖"
-   :message (:content msg)})
+  (let [message-reaction {:type :metabot.reaction/message
+                          :repl/message-color :green
+                          :repl/message-emoji "🤖"
+                          :message (:content msg)}
+        navigate-reaction (when-let [nav-path (:navigate-to msg)]
+                            {:type :metabot.reaction/redirect
+                             :url (str nav-path)})]
```

Comment:
Why not just 
```suggestion
                             :url nav-path})]
```

Reply:
Because Thomas is bad at Clojure 😆 . I'll adjust it.